docs/docs/forms.mdx

```diff
@@ -40,6 +40,32 @@ forms:
         entity: number
 ```
 
+There is also an optional parameter that can be defined in each form which is called
```

Comment:
aren't there other examples in the docs which we'll have to adapt to the new format as well?

Reply:
@wochinge So, I need to update all the examples that include `forms` : 
- in the docs, e.g. [from_entity](https://rasa.com/docs/rasa/forms#from_entity)
- maybe also in the [examples](https://github.com/RasaHQ/rasa/blob/dcaa9210326d41c7569fe5a0478cbc0d2f8ffb27/examples/formbot/domain.yml#L92) folder 
- and in rasa/data folder ? (eg. [default_with_slots.yml](https://github.com/RasaHQ/rasa/blob/dcaa9210326d41c7569fe5a0478cbc0d2f8ffb27/data/test_domains/default_with_slots.yml#L36))
- and wherever I find `forms` related examples

and add the keyword `required_slots` before the definition of slot mappings ?

Reply:
- So, I need to check if there are other places in the docs where forms are mentioned and check if I should mention also the `global_not_intent` parameter ? 
- Also, do I have to add the `required_slots` keyword : 
   - in all the `form` related examples in the docs ?
   - Also, in the [examples](https://github.com/RasaHQ/rasa/tree/e2759589e308de1120364035256d6169776c8c55/examples) folder ?
   - Maybe also in the examples in the tests, eg. [here](https://github.com/RasaHQ/rasa/blob/e2759589e308de1120364035256d6169776c8c55/tests/core/actions/test_forms.py#L76) ?

Reply:
I think all docs examples should use the `required_slots` key from now onwards. The old syntax should still work but give deprecation warnings. And yep, that would require changes in all of the mentioned locations 💯